examples/Makefile

```diff
@@ -17,7 +17,7 @@ LIBSSL_DIR = $(dir $(shell find ${BUILD_DIR} -name libssl.a))
 
 LDFLAGS = -L$(LIBCRYPTO_DIR) -L$(LIBSSL_DIR) -L$(LIB_DIR)
 
-LIBS = -l:libquiche.a -lev -ldl -pthread
+LIBS = -lquiche -lev -ldl -pthread
```

Comment:
`-l:libquiche.a` is not understood in MacOS linker but it works with Linux.

If the goal is to build with libquiche.a always, I prefer to use

`
-LIBS = -l:libquiche.a -lev -ldl -pthread
+LIBS = $(LIB_DIR)/libquiche.a -lev -ldl -pthread
`

To work with both platform.

Reply:
I mistakenly pushed changes (and reverted). my bad